pyparsing/core.py

```diff
@@ -2036,7 +2042,14 @@ def create_diagram(
                 "and jinja2 from https://pypi.org/project/jinja2 to generate parser railroad diagrams"
```

Comment:
As with the Travis, I would prefer that we recommend that the users install the extra instead of the packages separately, as written in the docs: https://pyparsing-docs.readthedocs.io/en/latest/HowToUsePyparsing.html#id20

Reply:
Yes! Will do.